templates/tracer_app.html

```diff
@@ -10,6 +10,7 @@
    -- analyze.html      --- the actual application!
 -->
 
+    <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
```

Comment:
So, this is not the only change to tracer_app.html you need for the graphs to appear. The commit does indeed restore the Download button but doesn't have the HTML elements to support the graphs.

Here's a picture of me locally running the current HEAD commit of the new_graph_branch branch:

![Screenshot 2024-10-08 at 8 21 36 AM](https://github.com/user-attachments/assets/120ded81-bbca-440d-bbdb-ad974eeaafc9)